src/components/BottomBar/SimulationIncrement.jsx

```diff
@@ -194,14 +277,24 @@ const mapStateToProps = (state) => {
     deltaTime: state.time.deltaTime,
     targetDeltaTime: state.time.targetDeltaTime,
     isPaused: state.time.isPaused,
-    luaApi: state.luaApi
+    luaApi: state.luaApi,
+    hasNextDeltaTimeStep: state.deltaTimes.hasNextDeltaTimeStep,
```

Comment:
I'd suggest to have luaApi last. Since this is tightly knit together with the time, it makes more sense to put it directly after isPaused.